github.user.styl

```diff
@@ -950,38 +952,64 @@ for c, i in C {
 
         &[type='radio'],
         &[type='checkbox'] {
-            &, &:focus, &:hover:active, &:disabled, &:hover:active:disabled {
+            &, &:focus, &:hover:active {
                 -webkit-appearance: none
                 -moz-appearance: none
                 appearance: none
                 cursor: pointer
-                height: 1rem
-                width: 1rem
+                height: b_hgwd
+                width: b_hgwd
                 border: 1px solid bg5
                 background-size: cover
                 c: fg1 0 bg1
             }
+            &:disabled, &:hover:active:disabled {
+                c: fg1 0 bg4
+            }
         }
 
-        &[type='radio']:checked {
-            z = '<circle cx="256" cy="256" r="128" fill="white"/>'
-            background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
-            border-radius: 100% i
-            &:checked { c: 0 ac4 ac4 }
+        &[type='radio'] {
+            /// fix border radius at https://github.com/settings/profile Emoji color
+            border-radius: 50%
+            &:checked {
+                z = '<circle cx="256" cy="256" r="128" fill="white"/>'
+                background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
+                c: 0 ac2 bg2
+            }
+            &:focus { border-radius: 50% }
+            &:hover { c: 0 ac1 bg2 }
         }
 
-        &[type='checkbox']:checked {
-            z ='<path fill="white" d="M99 213l-1 89 104 102 205-201v-91L203 315 99 213z"/>'
-            background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
-            c: 0 ac4 ac4
+        &[type='checkbox'] {
+            border-radius: 2px
+            &:checked {
+                z ='<path fill="white" d="M99 213l-1 89 104 102 205-201v-91L203 315 99 213z"/>'
+                background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
+                c: 0 ac2 bg1
+            }
+            &:hover { c: 0 ac1 bg2 }
         }
 
         &[type='checkbox']:indeterminate {
-            z = '<rect x="128" y="192" width="256" height="96" fill="white"/>'
+            z = '<rect x="120" y="120" width="256" height="256" fill="white"/>'
             background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
             c: 0 v16 v16
         }
     }
+    if (chbx_a_header) {
+        .Box-header {
+            & label {
```

Comment:
Unnecessary [parent reference][1] here and few lines below.

`.Box-header { & label { ... }}` == `.Box-header { label { ... }}`

[1]: https://stylus-lang.com/docs/selectors.html#parent-reference

Reply:
suggestions welcome

Reply:
Ive reworked this. need to rebase, deal with conflicts and push changes.